module/utils/species-command-parser.js

```diff
@@ -164,7 +170,7 @@ Hooks.on("dropCanvasData", (canvas, update) => {
         if(item) update.item = item;
     }
     if(update.item)
-        new game.ptu.PTUDexDragOptions(update, {"submitOnChange": false, "submitOnClose": true}).render(true);
+        new game.ptu.PTUDexDragOptions(update, {"submitOnChange": false, "submitOnClose": false}).render(true);
```

Comment:
@GSV-a-Nest-of-Kindled-Embers why this change? if a user presses enter/esc/close on this box it is most likely to skip the options, not to cancel generation. 

```suggestion
        new game.ptu.PTUDexDragOptions(update, {"submitOnChange": false, "submitOnClose": true}).render(true);
```

Reply:
When I was working on this and testing, I closed the popup and saw that it still generated, and at first I worried that _I_ had introduced a bug somehow, and afterwards just assumed it was just a pre-existing bug. Hitting close or ESC and the operation not cancelling is definitely not what I would think of as expected behavior, IMO.

Reply:
I _do_ want users to be able to just hit **Enter** and skip the options to continue generation the already-filled defaults, but I'm not sure I was able to get that working consistently with the `autofocus` tag on the submit button.